kernel/src/fs/mod.rs

```diff
@@ -67,4 +68,121 @@ pub fn lazy_init() {
         println!("[kernel] Mount ExFat fs at {:?} ", target_path);
         self::rootfs::mount_fs_at(exfat_fs, &target_path).unwrap();
     }
+
+    #[cfg(feature = "raid_test")]
+    init_raid1_test_device();
```

Comment:
If possible, let's make this consistent with all the other tests. So that means always initializing the device and having the test run from userspace. Does Asterinas have support for actual block device files in `/dev`? If so you should be able to to do the test pattern thing from a userspace program that is included in the initramfs. Then we can add a test mode, probably something to include all our userspace tests.

Reply:
However, it might not be possible. The issue is that ktests don't actually run in the full kernel, instead they run in a minimal test environment. I'm open to various approaches, but using a feature flag is really heavy weight and a misuse of feature flags.

Reply:
I'm not sure what you mean. Do you mean the test would be called whenever the kernel boots? That's not good. If you just mean you are initializing, the RAID device at boot, that's fine though.